lldb/tools/lldb-dap/OutputRedirector.cpp

```diff
@@ -36,7 +38,8 @@ Expected<int> OutputRedirector::GetWriteFileDescriptor() {
   return m_fd;
 }
 
-Error OutputRedirector::RedirectTo(std::function<void(StringRef)> callback) {
+Error OutputRedirector::RedirectTo(std::FILE *override,
```

Comment:
Let's avoid using a keyword.
```suggestion
Error OutputRedirector::RedirectTo(std::FILE *file_override,
```